apps/bandwagon/forms.py

```diff
@@ -136,6 +136,12 @@ class CollectionForm(ModelForm):
     icon = forms.FileField(label=_lazy(u'Icon'),
                            required=False)
 
+    # This is just a honeypot field for bots to get caught
+    your_name = forms.CharField(
+        label=_lazy(
+            u"Please don't fill out this field, it's used to catch bots"),
```

Comment:
If you add a comment above this formatted like `# L10n: bots is short for robots` then the localizers will see it in their web interface. It might help them out since English colloquialism can trip them up.